.github/workflows/publish-dispatch-prod.yml

```diff
@@ -0,0 +1,42 @@
+name: Dispatch Render for Juno Prod
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Enter image tag to be deployed'
+        required: true
+
+env:
+  EVENT_NAME: juno-prod
+  GOERLI: apps/juno-prod/overlays/prod-goerli-1/config.yaml
+  GOERLI2: apps/juno-prod/overlays/prod-goerli-2/config.yaml
+  TESTNET: apps/juno-prod/overlays/prod-integration/config.yaml
+  MAINNET: apps/juno-prod/overlays/prod-mainnet/config.yaml
+
+permissions:
+  id-token: write
+  contents: write
+
+jobs:
+  deploy:
+      environment: production
+      runs-on: ubuntu-latest
+
+      steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        
+  
+      - name: Repository Dispatch
```

Comment:
What are we triggering here? Is this related to a k8 deployment? `Repository Dispatch` doesn't provide much context. Can we consider a more descriptive name?